### PR TITLE
feat: adding links to mac and windows

### PR DESCRIPTION
**Pull request by tomobre**

### CODE REVIEW

**Review by yougotwill — CHANGES_REQUESTED** [review]
Very nice work man! Couple of changes need to be made. You can do it on Monday and Audric will review it before you merge it into master.

File: services/redirect.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  let link;
+  let hasPlatformLink = data.assets.some(
+    (el: { browser_download_url: string; content_type: string }) => {
+      if (platform === "/mac") {
+        link = el.browser_download_url;
+        return (
+          el.content_type === "application/x-xar" ||
+          el.content_type === "application/octet-stream"
+        );
+      } else {
```
Comment (majestrate) [inline services/redirect.ts:45]: platform is passed in as user input via a query paramter.
would suggest an else if windows and an else that will go to github

e.g. url given was /download?platform=solaris

File: services/redirect.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  permanent: boolean;
+}
+
+const redirects: IRedirection[] = getConfig().serverRuntimeConfig.redirects;
+
+async function fetchLatestVersionLink(repo: string, platform: string) {
+  const fallbackVersionLink = `https://github.com/oxen-io/lokinet/releases/download/${
+    platform === "/windows"
+      ? "v0.9.7/lokinet-0.9.7-win64.exe"
+      : "v0.8.4/lokinet-v0.8.4-macos.pkg"
```
[the remark this answers is not in the document]
Comment (majestrate) [reply services/redirect.ts:16]: may be better to use a map for the fallback links so there are more than just macos and windows

```
const githubFallbackLink = "https://github.com/oxen-io/lokinet";

const fallbackVersionLinks = {
   "/windows": "value here",
   "/mac": "value here",
   "/linux": "value here"
};

const fallbackVersionLink = fallbackVersionLink[platform] || githubFallbackLink;
```

File: components/DownloadLokinet.tsx [hunk 3]
```diff
@@ -37,6 +37,28 @@ const DownloadIconButton = styled.button`
...
+    <DownloadIconButton>
+      <div
+        onClick={() =>
+          window.open(
+            "https://docs.oxen.io/products-built-on-oxen/lokinet/guides/macos-gui-install-guide"
+          )
+        }
+      >
+        <Link href="mac">
+          <a rel="noopener noreferrer">
```
Comment (majestrate) [inline components/DownloadLokinet.tsx:51]: does CSP on lokinet.org permit inline svg?

Comment (Bilb) [reply components/DownloadLokinet.tsx:51]: I guess so. Here they are https://github.com/oxen-io/lokinet-website/commit/855a43e16c17f03a4e95e6165c9a0440b6d60030#diff-882b2c04b01b2e8b2cdcf1817c30ea503a8005f1c0d54cfff37053b6801fea85R1-R15

File: services/redirect.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bilb) [reply services/redirect.ts:16]: there is no binary redirect for linux as there are two many linux options. The button for linux just opens in a new tab https://docs.oxen.io/products-built-on-oxen/lokinet/guides/linux-gui-install-guide with details for each platform

File: services/redirect.ts (hunk 1 above)
Comment (Bilb) [reply services/redirect.ts:45]: no need for this as we do not support it. See my comment above about linux redirects

File: services/redirect.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bilb) [reply services/redirect.ts:16]: removing the redirect from `next.config.js`
 too

**Review by Bilb — APPROVED** [review]

**Pull request merged**